src/styles/header.scss

```diff
@@ -0,0 +1,26 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  max-width: 1200px;
```

Comment:
To avoid such view on wide screen remove `max-width`
![2023-10-04_16h15_14](https://github.com/mate-academy/layout_catalog/assets/61904995/b75a344c-e496-4b99-bb8b-4d878bed6967)